jobs/JREGIONAL_GET_EXTRN_MDL_FILES

```diff
@@ -210,6 +215,7 @@ cd_vrfy "${extrn_mdl_staging_dir}"
 get_extrn_mdl_file_dir_info \
   extrn_mdl_name="${EXTRN_MDL_NAME}" \
   anl_or_fcst="${anl_or_fcst}" \
+  retro_or_realtime="${retro_or_realtime}" \
```

Comment:
Is it a must to introduce a new variable "retro_or_realtime"? It looks like we can use "DO_RETRO" directly.

Reply:
I am not sure we want to use DO_RETRO directly.

Reply:
We may have more situation to identify in finding ics and lbcs. This can set many values for future check. Hoe new system will change the way to find the data.